code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -1336,3 +1336,782 @@
 		return
 
 	G.tail_storage.open(usr)
+
+//Hivenet Admin
+/mob/living/carbon/human/proc/hivenet_transmit()
+	set name = "Emergency Hivenet Transmission"
+	set desc = "Send a direct Hivenet transmission to your superiors in the Hive. Only to be used in dire circumstances, on pain of severe consequences.."
+	set category = "Hivenet"
+	var/hives = list("Zo'ra", "K'lax", "C'thur")
+
+	if(!(all_languages[LANGUAGE_VAURCA] in src.languages))
```

Comment:
All of the verbs in this file need to check for incapacitation. Most of them also don't have proper spacing between the verb setup (set name, set desc, set category) and the actual code - please add that.